remote/src/main/java/com/example/appengine/remote/RemoteApiSeedPageViewData.java

```diff
@@ -0,0 +1,215 @@
+package com.example.appengine.remote;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.GeoPt;
+import com.google.appengine.tools.remoteapi.RemoteApiInstaller;
+import com.google.appengine.tools.remoteapi.RemoteApiOptions;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class RemoteApiSeedPageViewData {
```

Comment:
Misc best practice: Declare a private default constructor, to prevent outsiders from constructing this, and to express that it's not mean to be used as an object you construct.